apps/music/style/music.css

```diff
@@ -193,6 +193,11 @@ body.tiles-mode #views {
   transform: translateX(0%);
 }
 
+body.tiles-mode #views-tiles,
+body.search-from-tiles-mode #views-tiles {
```

Comment:
@eeejay When searching from tiles mode, I think `#views-tiles` should be hidden because we will only display the search results.